hbase-server/src/main/java/org/apache/hadoop/hbase/master/MasterRpcServices.java

```diff
@@ -1697,7 +1697,6 @@ public UnassignRegionResponse unassignRegion(RpcController controller,
     try {
       final byte [] regionName = req.getRegion().getValue().toByteArray();
       RegionSpecifierType type = req.getRegion().getType();
-      final boolean force = req.getForce();
```

Comment:
we should add a comment in the protobuf defition for `UnassignRegionRequest` in `Master` that the field for force is ignored.

Reply:
Will fix later. Thanks.